src/frontend/parallelization_phase.nit

```diff
@@ -95,11 +107,24 @@ end
 		assert nullreturn isa AExpr
 		mainfun.n_block.as(ABlockExpr).n_expr.add(nullreturn)
 
+
 		# Create new body for the annotated fun
-		var s_newbody ="""
+		var s_newbody : nullable String = null
```

Comment:
I feel like i'm repeating myself but `s_newbody` does not need to be nullable or have an explicit type.